src/OrchardCore/OrchardCore.BackgroundTasks.Abstractions/BackgroundService.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Extensions.Hosting
+{
+    /// <summary>
+    /// Base class for implementing a long running <see cref="IHostedService"/>.
+    /// </summary>
+    public abstract class BackgroundService : IHostedService, IDisposable
```

Comment:
Should this be internal and in an Orchard-specific namespace? It might cause trouble for folks migrating to 2.1.

Reply:
Yes, this file is intended to be removed but you're right if someone migrate to 2.1